src/dotnet/LocalizableStrings.cs

```diff
@@ -1,30 +1,180 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Microsoft.DotNet.Tools
+﻿namespace Microsoft.DotNet.Tools
 {
-    internal static class LocalizableStrings
+    internal class LocalizableStrings
```

Comment:
It doesn't really matter, but I'm curious if there was a reason  for dropping static here.